libr/core/cmd_anal.c

```diff
@@ -5776,14 +5776,76 @@ static void cmd_anal_graph(RCore *core, const char *input) {
 	switch (input[0]) {
 	case 'f': // "agf"
 		switch (input[1]) {
+		case 0:// "agf"
+			r_core_visual_graph (core, NULL, NULL, false);
+			break;
+		case 'v':// "agfv"
+			eprintf ("\rRendering graph...");
+			r_core_visual_graph (core, NULL, NULL, 1);
+			r_cons_show_cursor (true);
+			break;
 		case 't':// "agft" - tiny graph
 			r_core_visual_graph (core, NULL, NULL, 2);
 			break;
-		case 0:
-			r_core_visual_graph (core, NULL, NULL, false);
+		case 'd':// "agfd"
+			if (input[2] == 'm') {
+				r_core_anal_graph (core, r_num_math (core->num, input + 2),
+					R_CORE_ANAL_GRAPHLINES);
+			}
+			else {
+				r_core_anal_graph (core, r_num_math (core->num, input + 1),
+					R_CORE_ANAL_GRAPHBODY);
+			}
+			break;
+		case 'j':// "agfj"
+			r_core_anal_graph (core, r_num_math (core->num, input + 1), R_CORE_ANAL_JSON);
+			break;
+		case 'J':// "agfJ"
+			r_core_anal_graph (core, r_num_math (core->num, input + 1),
+				R_CORE_ANAL_JSON | R_CORE_ANAL_JSON_FORMAT_DISASM);
+		case 'g':// "agfg"
+			if (input[2] == 'f') {
+				//TODO
+			}
+			RAnalFunction *fcn = NULL;
```

Comment:
use ternary operator instead of assigning to null and such. also the only thing that changes is the off_fcn, so you dont need to copy the r_anal-get_fcn_in call twice